.gitignore

```diff
@@ -68,9 +68,6 @@ target/
 # IPython Notebook
 .ipynb_checkpoints
 
-# pyenv
-.python-version
```

Comment:
Don't think this should be ignored. That would make it more difficult to update the version later.